satpy/readers/fci_l2_nc.py

```diff
@@ -240,8 +240,10 @@ def _get_proj_area(self, dataset_id):
         """Extract projection and area information."""
         # Read the projection data from the mtg_geos_projection variable
         a = float(self._projection.attrs['semi_major_axis'])
-        rf = float(self._projection.attrs['inverse_flattering'])
         h = float(self._projection.attrs['perspective_point_height'])
+        # FIXME: Some L2PF test data files have a typo in the keyname for the inverse flattening parameter. Use a
```

Comment:
I think you can remove the FIXME tag, because this is nothing that can be fixed now. So you could create an issue for this to make sure you remember.

Reply:
Fixed. I also filed an [issue](https://github.com/pytroll/satpy/issues/2051). Can you please assign it to me?

Reply:
Done, thanks